src/components/ClauseComponent.js

```diff
@@ -20,6 +20,10 @@ import {
 
 /* Components */
 import ConditionalBoolean from './ConditionalBoolean';
+import ClauseVariableList from './ClauseVariableList';
+
+/*slate-command */
+import regenerateKey from '../SlateCommands/RegenrateAllNodeKeys'
```

Comment:
There is a typo in the file name, presumably this should be `RegenerateAllNodeKeys`

Reply:
ooh, yeah, I didn't notice that, will fix it